src/test/java/io/stargate/sgv2/jsonapi/api/v1/InsertIntegrationTest.java

```diff
@@ -100,16 +100,16 @@ public void insertDocument() {
     public void insertDocumentWithDateValue() {
       String json =
           """
-        {
```

Comment:
Not done on purpose but `./mvnw fmt:format` sometimes/somehow seems to decide it wants to reformat. :-(